karma.conf.js

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable import/no-commonjs */
 
+const babel = require('rollup-plugin-babel');
```

Comment:
nit: order only changed

Reply:
I left this change as well because it also alphabetizes it, but can revert if you don't think that's a good enough reason to change it

Reply:
its good enough